gulpfile.js

```diff
@@ -0,0 +1,67 @@
+const gulp = require('gulp');
+
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+const browser = require('browser-sync');
+const sass = require('gulp-sass'); 
+const eslint = require('gulp-eslint');
+require('dotenv').config();
+
+
+gulp.task('nodemon', () => {
+  nodemon({
+    script: 'server.js'
+  , ext: 'js'
+  , env: { 'NODE_ENV': 'development' }
+  })
+});
+
+gulp.task('server', ['nodemon'], () => {
+    browser.init({
+        proxy: "http://localhost:6000",
+        port: 5000,
+        files: ["public/**/*.*"],
+        reloadOnRestart: true
+    });
+});
+ 
+gulp.task('test', () =>
+    gulp.src('test/**/*.js', {read: false})
+        .pipe(mocha({reporter: 'nyan'}))
+);
+
+gulp.task('sass', () => {
+  return gulp.src('./public/css/common.scss')
+    .pipe(sass())
+    .pipe(gulp.dest('./public/css'))
+});
+gulp.task('bower', () => {
+    bower()
+        .pipe(gulp.dest('./public/lib/'));
+});
+
+gulp.task('lint', () => gulp.src([
+    'gruntfile.js', 
```

Comment:
no need to lint gruntfile
if you are watching this, _'public/js/**/*.js'_, no need to watch _'public/js/*.js'_